common/lsp/message-stream-splitter.cc

```diff
@@ -55,7 +58,10 @@ int MessageStreamSplitter::ParseHeaderGetBodyOffset(absl::string_view data,
   }
 
   size_t end_key = found_ContentLength_header + kContentLengthHeader.size();
-  if (!absl::SimpleAtoi(header_content.substr(end_key), body_size)) {
+  absl::string_view header_value = header_content.substr(end_key);
+  header_value =
+      header_value.substr(0, header_value.find_first_not_of("0123456789"));
```

Comment:
You might be able to us `std::isnum` for this.

Reply:
Refined it with a `std::find_if()` (but using ascii comparison of digits for readability. `std::isdigit` would be unclear if this is decimal or decimal+hex).